include/gmsh.tcc

```diff
@@ -11,11 +11,14 @@ void GMSH::get_vertices(const std::string& filename) {
 
   //! Number of vertices
   double nvertices = std::numeric_limits<double>::max();
-  const double toplines = 4;
+  const unsigned toplines = 4;
+  const unsigned vertex_array_size = 3;
```

Comment:
Is this dimension? then use `ndimension`. `vertex_array_size` doesn't mean anything.